ethcore/src/ethereum/ethash.rs

```diff
@@ -154,6 +176,17 @@ impl From<ethjson::spec::EthashParams> for EthashParams {
 			eip649_reward: p.eip649_reward.map(Into::into),
 			expip2_transition: p.expip2_transition.map_or(u64::max_value(), Into::into),
 			expip2_duration_limit: p.expip2_duration_limit.map_or(30, Into::into),
+			eosc_transition: p.eosc_transition.map_or(u64::max_value(), Into::into),
+			eosc_treasury_address: p.eosc_treasury_address.map_or_else(Address::new, Into::into),
+			eosc_treasury_reward: p.eosc_treasury_reward.map_or_else(Default::default, Into::into),
+			eosc_stake_address: p.eosc_stake_address.map_or_else(Address::new, Into::into),
+			eosc_stake_reward: p.eosc_stake_reward.map_or_else(Default::default, Into::into),
+			neweosc_transition: p.neweosc_transition.map_or(u64::max_value(), Into::into),
+			neweosc_pow_reward: p.neweosc_pow_reward.map_or_else(Default::default, Into::into),
+			neweosc_fund_address: p.neweosc_fund_address.map_or_else(Address::new, Into::into),
+			neweosc_fund_reward: p.neweosc_fund_reward.map_or_else(Default::default, Into::into),
+			neweosc_pos_address: p.neweosc_pos_address.map_or_else(Address::new, Into::into),
+			neweosc_pos_reward: p.neweosc_pos_reward.map_or_else(Default::default, Into::into),
```

Comment:
What is `neweosc`?

Reply:
@5chdn `neweosc` is our code name of Anti-inflation Hardfork on block num 150,000 please refer here https://github.com/eosclassic/go-eosclassic/issues/5